.travis.yml

```diff
@@ -52,18 +60,35 @@ before_install:
     export CONDARC=ci/condarc
 
 install:
+  - |
+    set -e
+    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
+      # Run 'xvfb' (on Linux only)
+      export DISPLAY=:99.0
+      sudo /sbin/start-stop-daemon --start --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1920x1200x24 -ac +extension GLX +render -noreset
+      sleep 3
+    fi
   - export GIT_FULL_HASH=`git rev-parse HEAD`
   - conda create -y -n testenv python=$TRAVIS_PYTHON_VERSION numpy=$NUMPY
   - conda activate testenv
-  - conda install -y scikit-beam -c nsls2forge
-  - conda update -y scikit-beam -c nsls2forge  # Update, because sometimes a very old version is installed :(
+  - conda install -y scikit-beam=0.0.23 -c nsls2forge
```

Comment:
Should it be `'scikit-beam>=0.0.23'` for future versions?

Reply:
It could be. I am not sure why the correct version is not getting installed by default. It used to install a correct version, then I had to update the installed version to get a latest one, then update stopped working so I had to specify the exact version. '>=' should work as well.